src/attributes/mutations.ts

```diff
@@ -145,7 +173,13 @@ export const useAttributeValueCreateMutation = makeMutation<
 export const attributeCreateMutation = gql`
   ${attributeDetailsFragment}
   ${attributeErrorFragment}
-  mutation AttributeCreate($input: AttributeCreateInput!) {
+  mutation AttributeCreate(
+    $input: AttributeCreateInput!
+    $firstValues: Int
+    $afterValues: String
+    $lastValues: Int
+    $beforeValues: String
```

Comment:
Do we really need to fetch values after attribute creation?
I believe that the only thing we need here is its ID to redirect user to the details view